webpack.config.js

```diff
@@ -40,7 +40,23 @@ module.exports = {
             test: /\.tsx?$/,
             loader: "ts-loader",
             exclude: /node_modules/,
-        }]
+        },
+        {
+            test: /\.css$/i,
+            use: [
+                "style-loader",
+                {
+                    loader: "css-loader",
+                    options: {
+                        modules: {
+                            mode: 'icss'
+                        }
+                    }
+                },
+                "postcss-loader"
```

Comment:
You're using the `postcss-loader` but `postcss` doesn't seem to be configured anywhere. (There's probably a warning to that effect being displayed when building.) In most projects we configure `postcss` to run `autoprefixer`.